### PR TITLE
Fix bugs

### PR DESCRIPTION
**Pull request by hshiah**
Fix bugs mentioned in google docs.
Add partial test code.

### CODE REVIEW

**Review by ryan1604 — APPROVED** [review]
LGTM

File: src/main/java/seedu/financialplanner/commands/ReminderListCommand.java [hunk 1]
```diff
@@ -14,7 +14,8 @@ public class ReminderListCommand extends Command {
     public static final String USAGE = "reminderlist";
     public static final String EXAMPLE = "reminderlist";
 
-    private static final java.util.logging.Logger logger = java.util.logging.Logger.getLogger("Financial Planner Logger");
+    private static final java.util.logging.Logger logger = java.util.logging.Logger.
```
Comment (ryan1604) [inline src/main/java/seedu/financialplanner/commands/ReminderListCommand.java]: can import the package then just use Logger logger instead

**Review by ryan1604 — CHANGES_REQUESTED** [review]

File: src/test/java/seedu/financialplanner/commands/AddReminderCommandTest.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+        try {
+            AddReminderCommand testEntry = new AddReminderCommand(
+                    Parser.parseRawCommand("addreminder /t    /d 11/12/2023"));
+            fail();
+        } catch (IllegalArgumentException e) {
+            assertEquals("Reminder type cannot be empty", e.getMessage());
+        }
+    }
+}
```
Comment (NeoMinWei) [inline src/test/java/seedu/financialplanner/commands/AddReminderCommandTest.java:54]: nice testing

**Review by NeoMinWei — APPROVED** [review]
LGTM

File: src/main/java/seedu/financialplanner/commands/ReminderListCommand.java [hunk 3]
```diff
@@ -14,7 +14,7 @@ public class ReminderListCommand extends Command {
     public static final String USAGE = "reminderlist";
     public static final String EXAMPLE = "reminderlist";
 
-    private static final java.util.logging.Logger logger = java.util.logging.Logger.
+    private static final Logger logger = java.util.logging.Logger.
```
Comment (ryan1604) [inline src/main/java/seedu/financialplanner/commands/ReminderListCommand.java]: Didnt change the second part

**Review by ryan1604 — APPROVED** [review]

**Pull request merged**